gwpy/signal/filter_design.py

```diff
@@ -306,7 +306,40 @@ def bilinear_zpk(zeros, poles, gain, fs=1.0, unit='Hz'):
     return dzeros, dpoles, dgain
 
 
-def parse_filter(args, analog=False, sample_rate=None):
+def convert_to_digital(form, filter, sample_rate):
+    """Convert an analog filter to digital via bilinear functions.
+
+    Parameters
+    ----------
+    form : `str`
+        type of filter, 'zpk' or 'ba'
+
+    filter: `tuple`
+        filter values
+
+    sample_rate: `float`
+
+    Returns
+    -------
+    dform : `str`
+        type of filter, 'zpk' or 'ba'
+
+    dfilter: 'tuple'
+        digital filter values
+    """
+
+    if form not in ('zpk', 'ba'):
+        raise ValueError(f"Cannot convert form {form} to digital. Only"
+                         f"'zpk' or 'ba' are allowed.")
+
+    if form == 'ba':
+        return 'ba', signal.bilinear(*filter, fs=sample_rate)
+
+    if form == 'zpk':
+        return 'zpk', bilinear_zpk(*filter, fs=sample_rate)
```

Comment:
It might be nice, if a little redundant, to have `convert_to_digital` call out to `parse_filter` to get things in a standard format, rather than requiring an extra argument, or introspecting the input object.